cli/load_to_mongo.py

```diff
@@ -184,7 +175,13 @@ def import_tags(drop=False):
 
 
 def list_batches():
-    return Batches.objects.order_by("created")
+    batch_counts = Shreds.objects.map_reduce(
+        map_f='function() { emit(this.batch, 1); }',
```

Comment:
http://docs.mongodb.org/manual/reference/operator/aggregation/group/#group-by-month-day-and-year

Probably combined with: http://docs.mongoengine.org/apireference.html?highlight=aggregate#mongoengine.queryset.QuerySet.aggregate